.travis.yml

```diff
@@ -38,9 +36,7 @@ jobs:
       addons:
         apt:
           packages:
-            - python3-gdal
             - python3-pycurl
-            - python3-psycopg2
       install:
         - pip install sphinx sphinx-rtd-theme
         - pip install -r requirements.txt
```

Comment:
Seems to be able to build the documentation even without installing the `requirements.txt`